storage/innobase/xtrabackup/test/t/keyring_pxb_1961.sh

```diff
@@ -0,0 +1,68 @@
+############################################################################
+# PXB-1961: xtrabackup fails with encryption='n' with Percona server
+############################################################################
+
+function backup_and_restore() {
+  record_db_state test
+  rm -rf $topdir/backup
+  mkdir -p $topdir/backup
+  # Backup
+  run_cmd xtrabackup --backup --target-dir=$topdir/backup --xtrabackup-plugin-dir=${plugin_dir}
+
+  # Prepare
+  run_cmd $XB_BIN $XB_ARGS --xtrabackup-plugin-dir=${plugin_dir} --prepare \
+  --target-dir=$topdir/backup ${keyring_args}
+
+  # Restore
+  stop_server
+  rm -rf $mysql_datadir
+  run_cmd xtrabackup --copy-back --target-dir=$topdir/backup
+  start_server
+  # Verify backup
+  verify_db_state test
+}
```

Comment:
this seems very useful. Can you make it a bit generic by passing database as input parameter.

and move it to common.sh

Reply:
@rahulmalik87 can you please address this comment as well?